settings.cfg.sh

```diff
@@ -8,6 +8,9 @@ LSSTSW_BUILD_DIR=${LSSTSW_BUILD_DIR:-${LSSTSW}/build}
 # lsstswBuild.sh
 # options passed to ./create_xlinkdocs.sh as cli options
 DOC_PUSH_PATH=${DOC_PUSH_PATH:-"$HOME/public_html/doxygen"}
+# lsstcam
+LSSTCAM_BUCKET=${LSSTCAM_BUCKET:-'rubin-ci-lsst/testdata_ci_lsstcam_m49'}
+LSSTCAM_TESTDATA_DIR=${LSSTCAM_TESTDATA_DIR:-'/tmp/LSSTCAMData'}
```

Comment:
The canonical spelling of the instrument's name has lowercase "am"; I think it might be better to use that.